modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -15,3 +15,8 @@ You can uninstall the {pipelines-title} Operator by using the *Administrator* pe
 . Click the *{pipelines-title}* Operator tile. The Operator tile indicates that the Operator is installed.
 
 . In the *{pipelines-title}* Operator description page, click *Uninstall*.
+
+[WARNING]
+====
+When you uninstall the Operator, you lose all the resources in the Pipeline namespace, including the secrets that you configured with it.
```

Comment:
I don't think the sentence needs the commas, based on https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas.

Reply:
+1 to Courtney. You could also do
```suggestion
When you uninstall the Operator, all resources in the Pipeline namespace are lost including the secrets you configured with it.
```

Reply:
Noted, I will rework it.

Reply:
Hi @opayne1, I have implemented and resloved the issue as suggested. Thank you.

/label peer-review-needed